test-suite/src/test/java/io/micronaut/microstream/docs/CustomerControllerTest.java

```diff
@@ -68,10 +69,13 @@ void verifyCrudWithMicroStream(String customerRepositoryImplementation) throws E
         assertEquals(sergioFirstName, customer.getFirstName());
         assertNull(customer.getLastName());
 
+        // Check the rest-api endpoint is working
+        client.retrieve("/microstream/object/1000000000000000031?valueLength=10000&variableLength=0&references=true");
+
         // When we restart the server
         httpClient.close();
         embeddedServer.close();
-        embeddedServer = ApplicationContext.run(EmbeddedServer.class, properties);
+        embeddedServer = ApplicationContext.run(EmbeddedServer.class, properties, Environment.TEST);
```

Comment:
Environment.TEST is not required. It is automatically inferred.

Reply:
https://github.com/micronaut-projects/micronaut-microstream/pull/96/commits/2e9eb9712abdc672fef6c37c1791a3259ca33d4a